src/content-service/models/venue.ts

```diff
@@ -15,7 +15,7 @@ export class Venue {
       name,
       address,
     } = venueData;
-    this.id = id;
+    this.id = id?.toString();
```

Comment:
Looks like input type for this parameter should also be updated

Reply:
it have valid type already 
`export interface ApiVenueData {
  id: string;
  name: string;
  address: ApiAddressData;
}`

Reply:
Ok, but you used number as a parameter during integration. Are you sure you updated the place which requires it as a number?